Dockerfile-coolify

```
@@ -0,0 +1,34 @@
+FROM node:20.18.0
+
+WORKDIR /app
+
+# Instalar pnpm e wrangler
+RUN corepack enable pnpm && \
+    npm install -g wrangler
```

to address problem https://github.com/stackblitz-labs/bolt.diy/commit/c88938cffcf09920933073663a4c5ffcc8689673
```suggestion
    npm install -g wrangler corepack@latest
```